packages/common/src/prompts/prompts.ts

```diff
@@ -207,9 +221,12 @@ const textareaPrompt = baseCustomPrompt.merge(promptValidationProps).extend({
   component: z.literal('textarea-prompt'),
 });
 
-const timePickerPrompt = baseCustomPrompt.merge(promptValidationProps).merge(timePicker).extend({
-  component: z.literal('time-picker-prompt'),
-});
+const timePickerPrompt = baseCustomPrompt
```

Comment:
survey frontend timepicker implementation is missing, only `meal-time-prompt` is in PR, can you please add?